doc/changelog.md

```diff
@@ -347,7 +386,7 @@
   qubit re-allocation.
   [(#473)](https://github.com/PennyLaneAI/catalyst/pull/473)
```

Comment:
@maliasadi what does this changelog entry mean?

Reply:
There was a bug in the implementation of `AllocateQubits` (batch qubit allocations) in which second calls to this method would create more than required runtime qubit IDs. Similar to PR #479, this fix doesn't have any direct user impact.